<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableEdit</class>
 <widget class="QWidget" name="TableEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>615</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>615</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>table edit</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(35, 35, 35);
}</string>
  </property>
  <widget class="QPushButton" name="delete_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: rgb(34, 34, 34);
    text-decoration: none;
    /* margin-top: 10px; */
    padding: 1px;
    text-transform: uppercase;
    transition: 0.5s;
    display: inline-block;
    position: relative;
	background-color:rgb(169, 255, 186);
	font-family: Plain,Helvetica Neue,Arial,Noto Sans,sans-serif;
	border-radius: 20px;
}

QPushButton:hover {
	border: rgb(255, 170, 0);
    /* transform: translateY(3px);*/
    /*transform: skewY(-1deg); */
    cursor: pointer;
    background-color: rgb(115, 255, 148);
}</string>
   </property>
   <property name="text">
    <string>complete the query</string>
   </property>
  </widget>
  <widget class="QPushButton" name="drop_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: rgb(34, 34, 34);
    text-decoration: none;
    /* margin-top: 10px; */
    padding: 1px;
    text-transform: uppercase;
    transition: 0.5s;
    display: inline-block;
    position: relative;
	background-color: rgb(255, 94, 97);
	font-family: Plain,Helvetica Neue,Arial,Noto Sans,sans-serif;
	border-radius: 20px;
}

QPushButton:hover {
	border: rgb(255, 170, 0);
    /* transform: translateY(3px);*/
    /*transform: skewY(-1deg); */
    cursor: pointer;
    background-color: rgb(255, 53, 56);
}</string>
   </property>
   <property name="text">
    <string>Drop table</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="query_lineedit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>570</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font-family: Plain,Helvetica Neue,Arial,Noto Sans,sans-serif;
	padding-left: 1em;
	color: white;
	background-color: rgb(104, 104, 104);
	border-radius: 20px;
	border:2px solid  rgb(104, 104, 104) ;
}

QLineEdit:hover {
	color: rgb(178, 246, 159);
    cursor: pointer;
    background-color: rgba(104, 104, 104, 0.911);
}

QLineEdit:focus {
	border-color:rgb(169, 255, 186);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>pattern name</string>
   </property>
  </widget>
  <widget class="QTableView" name="pattern_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>59</y>
     <width>570</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
	background-color: rgb(104, 104, 104);
	color: white;
	border-radius: 20px;
}



QHeaderView::section {
	background-color: rgb(104, 104, 104);
	color: white;
}
QHeaderView {
	background-color: rgb(104, 104, 104);
	color: white;
}
QTableCornerButton::section {
	background-color: rgb(104, 104, 104);
	color: white;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="table_edit_lable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-family: Plain,Helvetica Neue,Arial,Noto Sans,sans-serif;
	padding-left: 1em;
	color: white;
	background-color: rgb(104, 104, 104);
	border-radius: 20px;
	border:2px solid  rgb(104, 104, 104) ;
	text-transform: none;
}

QLabel:hover {
	color: rgb(178, 246, 159);
    cursor: pointer;
    background-color: rgba(104, 104, 104, 0.911);
}
</string>
   </property>
   <property name="text">
    <string>Insert name of the element  you want to delete below</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: rgb(34, 34, 34);
    text-decoration: none;
    /* margin-top: 10px; */
    padding: 1px;
    text-transform: uppercase;
    transition: 0.5s;
    display: inline-block;
    position: relative;
	background-color:rgb(169, 255, 186);
	font-family: Plain,Helvetica Neue,Arial,Noto Sans,sans-serif;
	border-radius: 20px;
}

QPushButton:hover {
	border: rgb(255, 170, 0);
    /* transform: translateY(3px);*/
    /*transform: skewY(-1deg); */
    cursor: pointer;
    background-color: rgb(115, 255, 148);
}</string>
   </property>
   <property name="text">
    <string>update table</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
